src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/SelfSignAccessRequirementWidget.java

```diff
@@ -15,6 +15,7 @@
 import org.sagebionetworks.web.client.SynapseJavascriptClient;
 import org.sagebionetworks.web.client.security.AuthenticationController;
 import org.sagebionetworks.web.client.utils.Callback;
+import org.sagebionetworks.web.client.widget.asynch.IsACTMemberAsyncHandler;
```

Comment:
Looks like this was based off of the other branch, pull in changes from the develop branch after these are added to develop through the original PR